src/interpolation.jl

```diff
@@ -0,0 +1,297 @@
+"""
+    nn_interpolation(...)
+
+* array_in is the array/map, which will be interpolated in a higher resolution.
+* nlongitude is the new width of the mesh, nlatitude will be calculated as nlongitude/2 + 1
+"""
+function nn_interpolation(array_in, nlongitude=256)
+    
+     (width_in, height_in) = size(array_in)
+     width_out = nlongitude
+     height_out = convert(Int64, width_out/2 + 1)
+     array_out = fill(0, (width_out, height_out))
+   
+     # calculate the new distance between each dot/pixle (ratio)
+     # "-1" because indicies starts by 1 in Julia
+     ratio_w = (width_in-1)/(width_out-1)
+     ratio_h = (height_in-1)/(height_out-1)
+
+     row = ones(Float64, width_out)
+     for i = 1 : length(row)         
+          r = row[i] + (i-1) * ratio_w
+          row[i] = r
+     end
+     row[end] = round(width_in) 
+    
+     col = ones(Float64, height_out)
+     for i = 1 : length(col)   
+          c = col[i] + (i-1) * ratio_h
+          col[i] = c
```

Comment:
```suggestion
          col[i] = col[i] + (i-1) * ratio_h
```
This avoids creating new variables